indexing-service/src/main/java/io/druid/indexing/common/task/Task.java

```diff
@@ -49,8 +49,10 @@
     @JsonSubTypes.Type(name = "index_hadoop", value = HadoopIndexTask.class),
     @JsonSubTypes.Type(name = "index_realtime", value = RealtimeIndexTask.class),
     @JsonSubTypes.Type(name = "noop", value = NoopTask.class),
-    @JsonSubTypes.Type(name = "version_converter", value = VersionConverterTask.class),
-    @JsonSubTypes.Type(name = "version_converter_sub", value = VersionConverterTask.SubTask.class)
+    @JsonSubTypes.Type(name = "version_converter", value = SegmentConverterTask.class), // Backwards compat
```

Comment:
Agree, but I think we should only have one standard way of calling a task, and deprecate others.